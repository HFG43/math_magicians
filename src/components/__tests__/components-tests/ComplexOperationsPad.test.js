import React from 'react';
import renderer from 'react-test-renderer';
import ComplexOperationsPad from '../../complexOperations';

it('Renders <ComplexOperationsPad /> correctly', () => {
  const handleClick = jest.fn();

  const operationsPad = renderer
    .create(<ComplexOperationsPad handleClick={handleClick} />)
    .toJSON();
  expect(operationsPad).toMatchSnapshot();
});
