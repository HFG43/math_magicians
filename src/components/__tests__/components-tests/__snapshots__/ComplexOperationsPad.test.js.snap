// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <ComplexOperationsPad /> correctly 1`] = `
<div
  className="calc_complex_operations_pad"
>
  <div
    className="calc_complex_operation"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
  </div>
  <div
    className="calc_complex_operation"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
  </div>
  <div
    className="calc_complex_operation"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      %
    </button>
  </div>
</div>
`;
